umpf

```diff
@@ -394,7 +394,7 @@ find_branch_rev() {
 				local choice=""
 				read -e -i n -p "Use ${branch} instead? [y/n]: " choice
 				if [ "${choice}" == "y" ]; then
-					reply="${branch}"
+					reply="$(${GIT} rev-parse "${branch}")"
```

Comment:
Locally my diff is:
```
diff --git a/umpf b/umpf
index b8c25aa..d307c5e 100755
--- a/umpf
+++ b/umpf
@@ -394,7 +394,7 @@ find_branch_rev() {
                                local choice=""
                                read -e -i n -p "Use ${branch} instead? [y/n]: " choice
                                if [ "${choice}" == "y" ]; then
-                                       reply="${branch}"
+                                       reply="$(${GIT} rev-parse "${branch}^{}" 2> /dev/null)"
                                fi
                        fi
                fi
 ```
 but I don't recall why I directed stderr to /dev/null. I think I just noted that the hashinfo was generated like that when running without the prompt.
Sorry for not pushing a PR myself. I actually forgot about this issue after finding it

Reply:
It's done that way further up in the function for the usual case, so it makes sense to do it here too. I am doing it the same as you did now.